inspirehep/modules/workflows/workflows/article.py

```diff
@@ -376,7 +380,8 @@ class Article(object):
                         # TODO: once legacy is out, this should become
                         # unconditional, and remove the SEND_TO_LEGACY_AND_WAIT
                         # steps
```

Comment:
The call to `prepare_update_payload` should die (looks like it was a primitive merger). And the call to legacy should become something similar to:
```python
SEND_TO_LEGACY_AND_WAIT = [
    send_robotupload(
        marcxml_processor=hep2marc,
        mode="replace"
    )
    IF_ELSE(
        article_exists,
        [], # Nothing to do
        [  
            wait_webcoll, # We still need to wait for webcoll feedback
        ]
    ),
]